scripts/requirements.txt

```diff
@@ -0,0 +1,3 @@
+PyYAML
+requests
+git+https://github.com/OpenBAS-Platform/release.git@main
```

Comment:
Master on this repo

Reply:
No, we're targeting the main branch of the release repo, it does not have a master branch as far as I know

Reply:
My bad